app/views/pages/_editor_liquid_help.html.erb

```diff
@@ -1,344 +1,75 @@
-        the link and the commit hash for that specific version. <br>
-        The format is <code>{% gist [gist-link]/[commit-hash] %}</code> <br>
-        e.g. <br>
-        <code>{% gist https://gist.github.com/suntong/3a31faf8129d3d7a380122d5a6d48ff6/f77d01e82defbf736ebf4879a812cf9c916a9252 %}</code></p>
-      </dd>
-      <dt><code>Specific Version File Embed</code></dt>
-      <dd>
-        <p>You can choose to embed a specific version of a gist file. All you need
-        the link, the filename and the commit hash for that specific version . <br>
-        The format is <code>{% gist [gist-link]/[commit-hash] file=[filename] %}</code> <br>
-        e.g. <br>
-        <code>
-          {% gist https://gist.github.com/suntong/3a31faf8129d3d7a380122d5a6d48ff6/f77d01e82defbf736ebf4879a812cf9c916a9252 file=Images.tmpl %}
-        </code></p>
-      </dd>
-    </dl>
-    <h3>GitPitch Embed</h3>
-    <p>All you need is the GitPitch link:</p>
-    <code>
-      {% gitpitch https://gitpitch.com/gitpitch/in-60-seconds %}
-    </code>
-    <h3>Video Embed</h3>
-    <p>All you need is the <code>id</code> from the URL.</p>
-    <ul>
-      <li><strong>YouTube:</strong> <code>{% youtube dQw4w9WgXcQ %}</code></li>
-      <li><strong>Vimeo:</strong> <code>{% vimeo 193110695 %}</code></li>
-      <li><strong>Twitch:</strong> <code>{% twitch ClumsyPrettiestOilLitFam %}</code></li>
-    </ul>
-    <h3>Medium Embed</h3>
-    <p>Just enter the full URL of the Medium article you are trying to embed.</p>
-    <code>{% medium https://medium.com/s/story/boba-science-how-can-i-drink-a-bubble-tea-to-ensure-that-i-dont-finish-the-tea-before-the-bobas-7fc5fd0e442d %}</code>
-    <h3>SlideShare Embed</h3>
-    <p>All you need is the SlideShare <code>key</code>:</p>
-    <code>{% slideshare rdOzN9kr1yK5eE %}</code>
-    <h3>CodePen Embed</h3>
-    <p>All you need is the full CodePen <code>link</code>, ending in the pen ID code, as follows:</p>
-    <code>{% codepen https://codepen.io/twhite96/pen/XKqrJX %}</code>
-    <dl>
-      <dt><code>default-tab</code></dt>
-      <dd>
-        Add default-tab parameter to your CodePen embed tag. Default to <i>result</i><br>
-        <code>{% codepen https://codepen.io/twhite96/pen/XKqrJX default-tab=js,result %}</code>
-      </dd>
-
-      <dt><code>preview</code></dt>
-      <dd>
-        <p>Change the codepen URL path from /pen/ to /pen/preview/ (or to /embed/preview/) to use codepen's Click-to-Load preview embed feature in the resulting iframe.</p>
-        <pre>{% codepen https://codepen.io/propjockey/pen/preview/dyVMgBg %}</pre>
-      </dd>
+    <h3>Supported URL Embeds</h3>
 
-      <dt><code>height</code></dt>
-      <dd>
-        <p>Add height parameter to your CodePen embed tag and to the resulting iframe itself. Defaults to <i>600</i></p>
-        <pre>{% codepen https://codepen.io/propjockey/pen/dyVMgBg height=300 %}</pre>
-      </dd>
+    <ul class="crayons-card grid gap-2 grid-cols-2 p-4 list-disc">
```

Comment:
@aitchiss @jeremyf I tried to render the partial `app/views/pages/_supported_url_embeds_list.html.erb` right here, but I got a Routing Error. I suspect it's because this partial does not have access to `app/views/pages`? 🤷🏾‍♀️ 

Help please! The same is true for the `app/views/pages/_supported_nonurl_embeds_list.html.erb` partial